wgengine/magicsock/magicsock.go

```diff
@@ -2238,6 +2238,14 @@ func (c *Conn) bindSocket(ruc *RebindingUDPConn, network string, curPortFate cur
 			continue
 		}
 		trySetSocketBuffer(pconn, c.logf)
+
+		if debugPMTUD() {
+			err = setDontFragment(pconn, network)
+			if err != nil {
+				c.logf("magicsock: unable disable fragmentation for %v port %d: %v", network, port, err)
```

Comment:
Tiny typo: "unable disable" -> "unable to disable"

Reply:
perhaps something like: `magicsock: set dontfragment failed: %v...`